<ui version="4.0" >
 <class>RefinementOptionsDialog</class>
 <widget class="QDialog" name="RefinementOptionsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Refinement Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QCheckBox" name="restrainCAsCheckBox" >
     <property name="text" >
      <string>Restrain CA's</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="restrainEndsCheckBox" >
     <property name="text" >
      <string>Restrain Ends</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="verboseMessagesCheckBox" >
     <property name="text" >
      <string>Verbose Messages</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="refineWhileFittingCheckBox" >
     <property name="text" >
      <string>Refine while fitting</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label_9" >
       <property name="text" >
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2" >
      <widget class="QLabel" name="label_10" >
       <property name="text" >
        <string>Relative weight</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <widget class="QLabel" name="label_11" >
       <property name="text" >
        <string>Target sigma/tolerance</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Bond:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2" >
      <widget class="QSlider" name="bondSlider" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="3" >
      <widget class="QDoubleSpinBox" name="bondSpinBox" >
       <property name="decimals" >
        <number>3</number>
       </property>
       <property name="maximum" >
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep" >
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Angle:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2" >
      <widget class="QSlider" name="angleSlider" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="3" >
      <widget class="QDoubleSpinBox" name="angleSpinBox" >
       <property name="decimals" >
        <number>3</number>
       </property>
       <property name="maximum" >
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep" >
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Plane:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2" >
      <widget class="QSlider" name="planeSlider" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="3" >
      <widget class="QDoubleSpinBox" name="planeSpinBox" >
       <property name="decimals" >
        <number>3</number>
       </property>
       <property name="maximum" >
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep" >
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Torsion:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2" >
      <widget class="QSlider" name="torsionSlider" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="3" >
      <widget class="QDoubleSpinBox" name="torsionSpinBox" >
       <property name="decimals" >
        <number>3</number>
       </property>
       <property name="maximum" >
        <double>20.000000000000000</double>
       </property>
       <property name="singleStep" >
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="5" column="3" >
      <widget class="QDoubleSpinBox" name="bumpSpinBox" >
       <property name="decimals" >
        <number>3</number>
       </property>
       <property name="maximum" >
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep" >
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="6" column="0" >
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Map:</string>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="6" column="1" colspan="2" >
      <widget class="QSlider" name="mapSlider" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="3" >
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>71</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Bumps:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2" >
      <widget class="QSlider" name="bumpSlider" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RefinementOptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RefinementOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
